modules/internal/ChapelSyncvar.chpl

```diff
@@ -835,9 +947,12 @@ module ChapelSyncvar {
 
   inline operator :(from, type t:_singlevar)
```

Comment:
Couldn't these use the `deprecated` keyword instead?  Same question about the sync versions as well

Reply:
You are changing it, though?  You added the compilerWarning lines instead of the deprecated keyword?  That's why I'm mentioning it . . .